recipe/meta.yaml

```
@@ -12,46 +12,61 @@ source:
     - 0001-fix-arrowfs-skip-arrow-fs-test-on-windows.patch
 
 build:
-  number: 0
+  number: 1
   script_env:
     - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
 
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pybind11 >=2.6                         # [build_platform != target_platform]
-    - setuptools_scm <7
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-  host:
-    - pybind11 >=2.6
-    - python
-    - pip
-    - setuptools_scm <7
-  run:
-    - python
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+outputs:
+  - name: python-{{ name|lower }}
+    script: build-duckdb.sh  # [unix]
+    script: build-duckdb.bat  # [win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - pybind11 >=2.6                         # [build_platform != target_platform]
+        - setuptools_scm <7
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - pybind11 >=2.6
+        - python
+        - pip
+        - setuptools_scm <7
+      run:
+        - python
+
+    test:
+      imports:
+        - duckdb
+      requires:
+        - pip
+        - pytest
+        - mypy
+        - fsspec
+        - google-cloud-storage
+        - pandas
+        # Remove again
+        - pytest<8
+      source_files:
+        - tools/pythonpkg/tests
+      commands:
+        - pip check
+        # ppc64le/aarch64 is emulated and fails to run the test suite here
+        # Skip slow tests to avoid memory issues in CI, skip stub tests that is fragile under new version of mypy
+        - pytest tools/pythonpkg/tests --ignore=tools/pythonpkg/tests/slow/ --ignore=tools/pythonpkg/tests/stubs/  # [not (ppc64le or aarch64)]
+  - name: {{ name|lower }}
+    build:
+      noarch: generic
+      skip: true  # [not linux64]
+      requirements:
```

@ytausch This is the actual issue 😂 
```suggestion
    requirements:
```

---

Oh no, yes xD